From f4bca5ba47f131c07a5f8e81106913fbea3e48d9 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 5 Sep 2018 11:28:45 +0200
Subject: [PATCH 0693/1051] arm64: dts: marvell: armada-37xx: reduce the ahci
 range

Addresses above 0xe0178 are used by the comphy driver and they are not
used for ahci related configuration. This change will allow to separate
ahci related range from comphy related range.

Also in Armada 37xx SoC the SATA Host controller consists of one SATA
port so the region above 0xe0180 is not relevant (0x180-0x1100 are
reserved for ports 1-31 according to AHCI spec.).

Change-Id: I24a77309379276bdc190964400ef8652bb8f51cb
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 6d217e57acdc..35aa1bc85e12 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -352,7 +352,7 @@
 
 			sata: sata@e0000 {
 				compatible = "marvell,armada-3700-ahci";
-				reg = <0xe0000 0x2000>;
+				reg = <0xe0000 0x178>;
 				interrupts = <GIC_SPI 27 IRQ_TYPE_LEVEL_HIGH>;
 				status = "disabled";
 			};
-- 
2.17.1

